tests/schema_utils.py

```diff
@@ -0,0 +1,22 @@
+import sys
+from pathlib import Path
+
+
+class SchemasDirectoryNotFound(Exception):
+    pass
+
+
+def get_schemas_base_path(lib_name: str = 'libero-schemas') -> str:
+    for path in sys.path:
+        if lib_name in path:
+            return path
+    raise SchemasDirectoryNotFound(
+        f'Unable to find {lib_name} directory. {lib_name} not in sys.path')
+
+
+def get_schema_file_path(name: str) -> str:
+    path = Path(get_schemas_base_path())
+    matches = [match for match in path.glob('**/%s' % name)]
```

Comment:
do you expect more than one match? I'd raise an exception if there are 2 matches as there are potentially two conflicting versions of the library around?

Reply:
In this case I don't but `glob` always returns a list of matches as more generic queries can return multiple items. The search will be contained within the `libero-schemas` "package". I like the idea of checking for multiple matches but should it be the job of this function to check if there are multiple files with the same name in the same "package"?

Reply:
If the responsibility of this function is throwing away all but the first matches, it can have a conservative behavior in signaling when this happens (with a warning log, or with an exceptions if it's a scenario that should not occur)

Reply:
updated.